docs/guides/fine_tuning.md

```diff
@@ -152,7 +152,35 @@ See the [Model Zoo](../../model_zoo) to see which models have fine-tuning suppor
 
 See [Integrations](../integrations.md) to see how to track fine-tuning metrics.
 
-Once the fine-tune is launched, you can also [get the status of your fine-tune](../../api/python_client/#llmengine.fine_tuning.FineTune.get). You can also [list events that your fine-tune produces](../../api/python_client/#llmengine.fine_tuning.FineTune.get_events).
+## Monitoring the fine-tune
+
+Once the fine-tune is launched, you can also [get the status of your fine-tune](../../api/python_client/#llmengine.fine_tuning.FineTune.get). 
+You can also [list events that your fine-tune produces](../../api/python_client/#llmengine.fine_tuning.FineTune.get_events).
+```python
+from llmengine import FineTune
+
+fine_tune_id = "ft-cabcdefghi1234567890"
+fine_tune = FineTune.get(fine_tune_id)
+print(fine_tune.status)  # BatchJobStatus.RUNNING
+print(fine_tune.fine_tuned_model)  # "llama-2-7b.700101-000000
+
+fine_tune_events = FineTune.get_events(fine_tune_id)
+for event in fine_tune_events.events:
+    print(event)
+# Prints something like:
+# timestamp=1697590000.0 message="{'loss': 12.345, 'learning_rate': 0.0, 'epoch': 0.97}" level='info'
+# timestamp=1697590000.0 message="{'eval_loss': 23.456, 'eval_runtime': 19.876, 'eval_samples_per_second': 4.9, 'eval_steps_per_second': 4.9, 'epoch': 0.97}" level='info'
+# timestamp=1697590020.0 message="{'train_runtime': 421.234, 'train_samples_per_second': 2.042, 'train_steps_per_second': 0.042, 'total_flos': 123.45, 'train_loss': 34.567, 'epoch': 0.97}" level='info'
+
+
+```
+
+The status of your fine-tune will give a high-level overview of the fine-tune's progress.
+The events of your fine-tune will give more detail, such as the training loss and validation loss at each epoch, 
+as well as any errors that may have occurred. If you encounter any errors with your fine-tune, 
+the events are a good place to start debugging. For example, if you see `Unable to read training or validation dataset`,
```

Comment:
Do we have a separate error for malformed files, e.g. files that don't have `prompt,response` headers?

Would it make sense to put these in a table?

Reply:
Discussed offline: `prompt,response` header validation would happen earlier, at `FineTune.create` time.